hasher-matcher-actioner/webapp/src/utils/TextFieldsUtils.jsx

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+ */
+
+import React, {useState, useEffect, useCallback} from 'react';
+import {Popover, OverlayTrigger} from 'react-bootstrap';
+import copy from 'clipboard-copy';
+
+function truncateString(input, maxLength = 25) {
+  if (input == null) {
+    return '';
+  }
+  if (input.length <= maxLength) {
+    return input;
+  }
+  return `${input.substring(0, maxLength)}...`;
+}
+
+// eslint-disable-next-line react/prop-types
+export function CopyableHashField({text, tooltip}) {
```

Comment:
This seems like a great feature!

ymmv:
```suggestion
export function CopyableHashField({text, tooltip=''}) {
```

Reply:
why blank string?